lib/papertrail/http_client.rb

```diff
@@ -46,9 +46,8 @@ def token_auth(token)
     end
 
     def get(path, params = {})
-      if params.size > 0
-        path = "#{path}?cli_version=#{Papertrail::VERSION}&#{build_nested_query(params)}"
-      end
+      params.merge!(cli_version)
+      path = "#{path}?#{build_nested_query(params)}"
```

Comment:
My thought here would be that it used to be the case that we would add the version if there were params, but now we want to add it every time, so it can follow the same pattern as the other methods:

```
begin
  on_complete(https.get(request_uri(path), build_nested_query(params.merge(cli_version)), @headers))
rescue
```

Would that work, or is there something I'm missing?